src/frontend/Felles/Modal/ModalWrapper.tsx

```diff
@@ -35,40 +35,54 @@ const ModalKnapp = styled(Button)`
 interface ModalProps {
     tittel: string;
     visModal: boolean;
-    onClose: () => void;
+    closeButton?: boolean;
+    onClose?: () => void;
+    aksjonsknapper?: boolean;
     hovedKnappClick?: () => void;
-    lukkKnappDisabled?: boolean;
+    hovedKnappTekst?: string;
     hovedKnappDisabled?: boolean;
+    lukkKnappClick?: () => void;
     lukkKnappTekst?: string;
-    hovedKnappTekst?: string;
-    skjulKnapper?: boolean;
+    lukkKnappDisabled?: boolean;
+    maxWidth?: boolean;
+    ariaLabel?: string;
     children?: React.ReactNode;
 }
 
 export const ModalWrapper: React.FC<ModalProps> = ({
     tittel,
     visModal,
+    closeButton,
     onClose,
+    aksjonsknapper = true,
     hovedKnappClick,
-    lukkKnappDisabled,
+    hovedKnappTekst,
     hovedKnappDisabled,
+    lukkKnappClick,
     lukkKnappTekst,
-    hovedKnappTekst,
-    skjulKnapper,
+    lukkKnappDisabled,
+    maxWidth = true,
+    ariaLabel,
     children,
 }) => {
     return (
-        <ModalContainer open={visModal} aria-label={tittel} onClose={() => onClose()}>
+        <ModalContainer
+            open={visModal}
+            aria-label={ariaLabel ? ariaLabel : tittel}
+            onClose={onClose ? () => onClose() : () => null}
+            closeButton={closeButton}
```

Comment:
Her finnes en risiko at man har en closeButton, men som ikke gjør noe, burde man kanskje fjerne `closeButton: boolean`, å bruke `closeButton={!!onClose}`? 
Då har man lukkKnapp hvis man har definiert funksjon for den

Reply:
Fixd: [fdfb0b4](https://github.com/navikt/familie-ef-sak-frontend/pull/1641/commits/fdfb0b42cf1748adcc82e166ba79667a608a4e80)